src/java.base/share/classes/java/io/PrintStream.java

```diff
@@ -68,6 +68,7 @@
     private final boolean autoFlush;
     private boolean trouble = false;
     private Formatter formatter;
+    private Charset charset;
```

Comment:
Hello @naotoj, should this be formally marked as `final`?

Reply:
Good catch! I will make it `final`.